### ISSUE TITLE
Add a way to register template dashing.json files

### ISSUE THREAD

**Issue by mhintz** [technosophos/dashing#20]
My particular use case is generating Dash documentation for Rust crates. Since rust doc uses a standard structure, I have a common set of selectors and options that I want to use whenever generating these docs. It would be nice to have a way to register dashing.json files that could be used as optional defaults when running `dashing create`. For example:

`dashing create --template rustdoc`

Thanks for building a really useful tool!

**Comment by technosophos** [comment]
That's a really good idea. it'll be a while before I can get to it, though.

**Comment by mhintz** [comment]
Yeah, no worries! Just thought it would be a cool feature to see. I wish I knew more about Go, I’d put together a pull request.
